subgraphs/stakedao/src/Oracle/common/constants.ts

```diff
@@ -0,0 +1,135 @@
+import { Address, BigDecimal, BigInt, TypedMap } from "@graphprotocol/graph-ts";
+import { UniswapPair__getReservesResult } from "../../../generated/Controller/UniswapPair";
+import { SushiSwapPair__getReservesResult } from "../../../generated/Controller/SushiSwapPair";
+
+///////////////////////////////////////////////////////////////////////////
+//////////////////////////////////? COMMON ?///////////////////////////////

```

Comment:
nit: can you change `?` to `/` for consistency?

Reply:
I am using this extension https://marketplace.visualstudio.com/items?itemName=aaron-bond.better-comments which helps in writing better comments. If we can mainstream, then that would be great but I will remove this for now.

![image](https://user-images.githubusercontent.com/47497931/163121055-f15ee67c-763b-42a1-80b8-187dd5f18411.png)

![image](https://user-images.githubusercontent.com/47497931/163121314-5db38f6c-bf37-41c2-b874-c19315e3f0f1.png)